pkgs/applications/networking/mailreaders/evolution/evolution/wrapper.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, makeWrapper, symlinkJoin, adwaita-icon-theme, evolution, evolution-ews, gnome3 }:
+
+symlinkJoin {
+  name = "evolution-with-plugins";
+  paths = [ evolution evolution-ews gnome3.evolution-data-server];
+
+  buildInputs = [ makeWrapper ];
+
+  postBuild = ''
+    wrapProgram $out/bin/evolution \
+      --set LD_LIBRARY_PATH "$out/lib"
```

Comment:
Hmm, cool trick. That circumvents the nested patching issue. But we will need to make sure there are no side effects to child processes.

Reply:
Also this still does not deal with the fact that the desktop file points to the unwrapped evolution in `pkgs.evolution`.